content/recipes/interoperability/c4c-clinical-trials.md

```diff
@@ -0,0 +1,302 @@
+| status_recruitment | C4C Study Metadata Collection | Study Information | dropdown | Status: Recruitment |
+| study_documents | C4C Study Metadata Collection | Study Information | multiple choice | Study Documents Available |
+| study_documents\_other | C4C Study Metadata Collection | Study Information | short text | Add Additional Types of Study Documents |
+| study_results | C4C Study Metadata Collection | Study Information | dropdown | Study Results |
+| study_continents | C4C Study Metadata Collection | Study Information | multiple choice | Please Select Study Site Locations |
+| european_sites | C4C Study Metadata Collection | Study Information | multiple choice | Please Select European Study Site Locations |
+| n_american\_sites | C4C Study Metadata Collection | Study Information | multiple choice | Please Select North American Study Site Locations |
+|  | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | begin new section |  |
+| age | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | short text | Age Range |
+| age_group | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | multiple choice | Age Group(s) |
+| sex | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | dropdown | Sex |
+| ethnicity | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | multiple choice | Ethnicity |
+| inclusion_criteria | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | text box | Additional Inclusion Criteria |
+| exclusion_criteria | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | text box | Additional Exclusion Criteria |
+| outcome_measures | C4C Study Metadata Collection | Inclusion/Exclusion Criteria | text box | Outcome Measures |
+|  | C4C Study Metadata Collection | Treatment Information | begin new section |  |
+| intervention_treatment | C4C Study Metadata Collection | Treatment Information | ontology field | Intervention/Treatment |
+| orphan_designation | C4C Study Metadata Collection | Treatment Information | dropdown | IMP with orphan designation in the indication |
+| biospecimens_retained | C4C Study Metadata Collection | Treatment Information | dropdown | Biospecimens Retained |
+| biospecimens_text | C4C Study Metadata Collection | Treatment Information | text box | Type of Biospecimens Retained |
+| product_class | C4C Study Metadata Collection | Treatment Information | ontology field | Product Class |
+
+The REDCap survey will be sent to studies within the c4c consortium for additional testing. A representative of the study will be asked to complete the survey with metadata from their study and provide feedback. This feedback will be used to further refine the list of metadata items collected. The final metadata schema will be used to create a FAIR Data Point for c4c studies.  In the future this will allow researchers to find sources of paediatric data from clinical trials. 
```

Comment:
"The final metadata schema will be used to create a FAIR Data Point for c4c studies".
 Explain what is a FAIR Data Point and in what form the 'final metadata schema' should be ?

Reply:
Re: FAIR Data Point: an implementation of the [FAIR Data Point specifications](https://specs.fairdatapoint.org/), such as the [FAIR Data Point reference implementation](https://fairdatapoint.readthedocs.io/en/latest/index.html).

In such a FAIR Data Point, the metadata schema will be implemented using the Metadata schema editor of the reference implementation, resulting in a ShaCL representation of the schema.